src/Resources/model/Machine.py

```diff
@@ -58,6 +58,15 @@ def add_meta(self, name, value):
             self.bridge = self.lab.get_or_new_link(BRIDGE_LINK_NAME)
             return
 
+        if name == "sysctl":
+            # Check for valid kv-pair
+            if '=' in value:
+                parts = value.split('=')
+                # Only allow `net.` namespace with int value
+                if parts[0].strip().startswith('net.') and parts[1].strip().isdigit():
+                    self.meta['sysctls'][parts[0].strip()] = int(parts[1].strip())
```

Comment:
I've find out that there are some sysctls in the `net` namespace that are not assigned to a number but to a string.
For example on my machine I have `net.ipv4.tcp_allowed_congestion_control = reno cubic`.

So maybe it would be better to allow also strings after the `=`.

Reply:
I see, then I'll just auto-convert it instead of allowing